app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Expr.scala

```diff
@@ -45,6 +46,7 @@ sealed trait Expr extends Product with Serializable {
     case Else(field1: Expr, field2: Expr)           => field1.sums ++ field2.sums
     case FormCtx(formComponentId: FormComponentId)  => Nil
     case sum @ Sum(field1: Expr)                    => sum :: Nil
+    case Count(field1: FormComponentId)             => Nil
```

Comment:
**field1** never used.

Reply:
Yes, but it is consistent with rest of the definition. And someone can view it as a documentation for the reader.

If you are somehow concerned about doing that, let's take that to another discussion and let's do the change in one fell swoop across whole project

Reply:
I agree there are many places in the codebase so let it be inline.